report/main.qmd

```diff
@@ -0,0 +1,139 @@
+
+![**Sale of sugary and zero-calorie drinks by intervention**](../figs/eda-1_time-serie.png){#fig-1}
+
+In particular, @fig-1 shows some important characteristics of the dataset. Firstly, the measurements for each site began at different times. Secondly, the third site experienced a significant increase in sales during most of the first intervention, but afterwards, sales remained at a lower and more stable level. Thirdly, the order of the three calorie messaging interventions was different for each site. Lastly, it is evident from the data that there is a weekly seasonal effect.
+
+# Formal analysis
+
+To evaluate the research questions, it is suggested to utilize a preparation step along with two statistical methods. In the following subsections, the necessary data adjustments, a detailed explanation of each recommended model and the suggested procedure to conduct and analyze the results are presented.
+
+## Pre-analysis
+
+Before conducting the statistical analysis, it is important to address some elements that were identified during the Exploratory Analysis that could impact the results. These include the weekly effect, different lengths of the pre-intervention observations and unnecessary information. To focus on capturing the intervention effects instead of the number of days effects, one can eliminate the weekly impact by either using a percentage of sales or decomposing the time series, as suggested by @ChatfieldXing:2019. While the percentage approach may be easier, it is recommended to use the decomposition method as the variable containing the total items sold may have inconsistencies that suggest unreliable information.[^2]
+
+[^2]: For example, day 199 in the C site has more zero-calorie and sugary beverages sold than total items. Also, days 50 to 53, 107, 184 and 196 in the same site have decimals.
+
+As sites B and C have a short pre-intervention period, it is necessary to compensate for the missing information. This can be done by combining the follow-up and pre-intervention categories into a new category that will serve as the baseline for the study. Lastly, we are not interested in the washout periods that are used to reduce the effects of previous interventions. Hence, this data can be ignored.
+
+## ANOVA/ANCOVA
+
+Analysis of Variance (ANOVA) is a statistical method used to compare the means of three or more samples, determining if at least one sample mean significantly differs from the others. It's instrumental in experiments aimed at evaluating the effectiveness of different interventions. In this study, it is used to encourage the selection of zero-calorie beverages over sugary options in hospital settings. By examining the variance within and between intervention groups, ANOVA facilitates understanding whether observed differences in beverage consumption are attributable to the interventions or occur by chance.
```

Comment:
I am not sure whether ANOVA is informative here. The linear mixed model should also be able to tell the effectiveness of different interventions. 
Also, it seems inappropriate to combine the data during an intervention period for all sites, because of the obvious difference among sites. Maybe consider two-way ANOVA with one way on interventions and one way on sites?

Reply:
What questions are you hoping to address with this methodology? What are the necessary assumptions?